test/basic-auth-request-test.js

```diff
@@ -79,6 +79,39 @@ test('basic auth request with 401 error', (t) => {
   });
 });
 
+test('basic auth request with defined auth url', (t) => {
+  const basicAuthRequest = proxyquire('../lib/basic-auth-request', {
+    request: (requestObject, cb) => {
+      t.equal(requestObject.strictSSL, false, 'should be false');
```

Comment:
I think you need to change the test here to test the `requestObject.url` to make sure the authUrl you passed in is included in it

Reply:
OK good point. Thanks